.circleci/base_config.yml

```diff
@@ -205,10 +211,15 @@ jobs:
       - run:
           name: Print SCCache Settings
           command: sccache -s
+      - run:
+          name: Setup SCCache for clang
+          command: |
+            ln /usr/bin/sccache /usr/local/bin/clang
+            ln /usr/bin/sccache /usr/local/bin/clang++
       - run:
           name: Configure
           command: |
-            cmake --preset << parameters.preset >> -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_EXE_LINKER_FLAGS="-fuse-ld=lld" -DCMAKE_LIBRARY_PATH=$OPENSSL_ROOT_DIR/lib
+            cmake --preset << parameters.preset >> -DCMAKE_C_COMPILER=<< pipeline.parameters.c-compiler >> -DCMAKE_CXX_COMPILER=<< pipeline.parameters.cxx-compiler >> -DCMAKE_EXE_LINKER_FLAGS="-fuse-ld=lld" -DCMAKE_LIBRARY_PATH=$OPENSSL_ROOT_DIR/lib
```

Comment:
I am totally on your page.
We just want this as an intermediate solution until we can switch properly over to clang.

Replication 2 reliably hits the libunwind bug, that is not present in the CLANG compiled arangodb.
=> We have many suites that crash instead of doing proper tests which gives us huge blind spots on the feature.